pkg/controller/instance/controller_reconcile.go

```diff
@@ -44,6 +47,16 @@ const (
 	ResourceStatePendingDeletion     = "PENDING_DELETION"
 	ResourceStateWaitingForReadiness = "WAITING_FOR_READINESS"
 	ResourceStateUpdating            = "UPDATING"
+
+	FieldManagerForApplyset = "kro.run/applyset"
```

Comment:
the applyset code defines labels, are those not needed?

Reply:
This is for field manager in apply. Labels are orthogonal.